src/arc.ts

```diff
@@ -126,7 +134,7 @@ export async function initializeArc(provider?: any): Promise<boolean> {
     const retryLink = new RetryLink({
       attempts: {
         max: 5,
-        retryIf: (error, _operation) =>  {
+        retryIf: (error, _operation) => {
```

Comment:
why this change ? how the linter did not catch that before..is it catching that at all?
if linter does not care about so leave these update for other "lint": pr.. it is difficult to review with all these changes.

Reply:
1) On this github page you can tell the github to ignore whitespage differences
2) I already created a separate PR to make the linter catch these:  https://github.com/daostack/alchemy/pull/1650